java/test/android/controller/build.gradle

```diff
@@ -51,11 +51,11 @@ repositories {
 }
 
 android {
-    compileSdkVersion 33
+    compileSdk 33
 
     defaultConfig {
         applicationId "com.zeroc.testcontroller"
-        minSdkVersion 31
+        minSdkVersion 24
```

Comment:
why decrease this one?

Reply:
This is the version we support, as per the release notes, and I wanted to test it.